docker-environment-driver/src/test/java/org/jboss/pnc/environment/docker/DockerEnvironmentDriverRemoteTest.java

```diff
@@ -31,22 +15,36 @@
 import org.jboss.pnc.spi.repositorymanager.RepositoryManagerResult;
 import org.jboss.pnc.spi.repositorymanager.model.RepositoryConnectionInfo;
 import org.jboss.pnc.spi.repositorymanager.model.RepositorySession;
+import org.jboss.pnc.test.category.ContainerTest;
+import org.jboss.pnc.test.category.RemoteTest;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
-import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.jboss.shrinkwrap.resolver.api.maven.Maven;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
+import javax.inject.Inject;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.Socket;
+import java.util.Map;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.*;
+
 /**
  * Unit tests for DockerEnnvironmentDriver
- * 
- * @author Jakub Bartecek <jbartece@redhat.com>
  *
+ * @author Jakub Bartecek <jbartece@redhat.com>
  */
 @RunWith(Arquillian.class)
+@Category({RemoteTest.class, ContainerTest.class})
```

Comment:
Double checked - this was a container-based test before my changes (see [1]). It should behave exactly the same way after my changes - container-based test which connects to external resources. That's why we need to 2 profiles there.

[1] https://github.com/project-ncl/pnc/pull/243/files#diff-d67f3d8a4636c41bf582c63a0f397bc9L141